### PR TITLE
[WIP] Bundler API v1 Migration -- Part 2

### PR DESCRIPTION
**Pull request by maclover7**
r? @dwradcliffe

### CODE REVIEW

File: app/models/gem_dependent.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def fetch_dependencies
       end
     end
 
-    dependencies
+    dependencies.flatten
```
Comment (arthurnn) [inline app/models/gem_dependent.rb]: why this flatten here? what was this returning before?

Comment (maclover7) [inline app/models/gem_dependent.rb]: It was returning an array of arrays (one array per gem that was requested, and then wrapped up in a giant array after that) -- but this isn't what bundler.rubygems.org currently returns in production, so we need to make sure we match that 😬

**Comment by dwradcliffe** [comment]
@maclover7 Is this good to merge?

**Comment by maclover7** [comment]
@dwradcliffe either in this PR, or in another PR, we need to figure out how to handle and display the `required_rubygems_version` for gems. I know you cc'ed me on an issue, not sure how we want to move forward with that 😬

**Comment by dwradcliffe** [comment]
That's being worked on in #1276

**Pull request merged**